File: Homeworks/homework008.py
Car = {'brand': 'empty', 'gen': 'empty', 'color': 'empty', 'price': '0'}

Car['brand'] = input("Please input car brand: ")
Car['gen'] = input("Please input car generation: ")
Car['color'] = input("Please input car color: ")
Car['price'] = input("Please input car price: ")

print("---------------------------------------------")

print("Car brand: %s" %Car['brand'])
print("Car generation: %s" %Car['gen'])
print("Car color: %s" %Car['color'])
print("Car price: %s" %Car['price'])

File: Homeworks/homework008.cpp
#include <iostream>
using namespace std;

struct Car{
	char brand[50];
	char gen[50];
	char color[50];
	int price;
} myCar;

int main(){
	//input
	cout << "Please input car brand: ";
	cin >> myCar.brand;
	cout << "Please input car generation: ";
	cin >> myCar.gen;
	cout << "Please input car color: ";
	cin >> myCar.color;
	cout << "Please input car price: ";
	cin >> myCar.price;
	
	cout << "------------------------------------------------" << endl; 
	
	cout << "Car brand: " << myCar.brand << endl;
	cout << "Car generation: " << myCar.gen << endl;
	cout << "Car color: " << myCar.color << endl;
	cout << "Car price: " << myCar.price << endl;
}
